MIDAS/src/systems.cpp

```diff
@@ -165,6 +165,16 @@ DECLARE_THREAD(kalman, RocketSystems* arg) {
     }
 }
 
+/**
+ * See \ref data_logger_thread
+ */
+DECLARE_THREAD(telemetry, RocketSystems* arg) {
+    while (true) {
+        arg->tlm.transmit(arg->rocket_data);
+        THREAD_SLEEP(16);
```

Comment:
why 16ms + need to buffer data somewhere

Reply:
the 16 doesn't matter anyways since transmit has a 170 ms delay in it